src/app/services/itinerary.service.ts

```diff
@@ -11,15 +11,34 @@ export class ItineraryService {
 
   constructor(private httpClient: HttpClient) { }
 
-  hostURL = 'https://be-dev-yourney.herokuapp.com/';
+  // hostURL = 'http://localhost:3000/';
+  id = 1
+  hostURL = 'https://be-dev-yourney.herokuapp.com';
 
   public userItineraries(page:Number, userId:Number): Observable<ItineraryUserPage> {
-    var url=this.hostURL + 'itinerary/user/'+page;
+    var url=this.hostURL + '/itinerary/user/'+page;
     //const headers = { 'Authorization': 'Bearer my-token', 'My-Custom-Header': 'foobar' };
     var body_params = { "userId": userId };
     var req=this.httpClient.post<ItineraryUserPage>(url,body_params);
     //req.subscribe(res=>console.log(res))
     return req;
   }
 
+  public vista(): Observable<Itinerary> {
+    var url=this.hostURL + '/itinerary/show/' + this.id.toString()
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
    var url=this.hostURL + "/itinerary/show/" + this.id.toString()
```